frontend/src/components/institute/NewSchoolPage.vue

```diff
@@ -471,6 +478,14 @@ export default {
 
       this.fireFormValidate();
     },
+    constrainCategoryByDistrict(num) {
```

Comment:
Minor naming suggestion to make num => districtNumber. Easier to know what the function is asking for.